framework/Makefile

```diff
@@ -12,7 +12,7 @@ LOG_INSTALLDIR         ?= /var/log/${WAZUH_SERVER}
 LIB_INSTALLDIR         ?= /var/lib/${WAZUH_SERVER}
 ETC_INSTALLDIR         ?= /etc/${WAZUH_SERVER}
 RUN_INSTALLDIR         ?= /run/${WAZUH_SERVER}
-BIN_INSTALLDIR         ?= /bin
+BIN_INSTALLDIR         ?= /usr/share/wazuh-server/bin
```

Comment:
```suggestion
BIN_INSTALLDIR         ?= ${SHARE_INSTALLDIR}/bin
```

Reply:
Changed in b5a5da997b99742d75667000bfec999dd20878f8.